packages/flutter_tools/lib/src/runner/environment_variables.dart

```diff
@@ -0,0 +1,6 @@
+// this file contains the environment variables supported to configure Flutter
+// at runtime
+
+/// adjusts whether Flutter should run its version check when running the
+/// Flutter tool
+const String flutterVersionCheckName = 'FLUTTER_VERSION_CHECK';
```

Comment:
any new files must have license information at the top; however, i think this can be in the `flutter_command_runner.dart` file.

Reply:
also, constants should start with `k` https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k